File: Java/Hardware.java
package W2.Latihan1.Java;

class Hardware extends Product {
    private String Brand;
    private String Model;

    public Hardware(String Brand, String Model) {
        this.setBrand(Brand);
        this.setModel(Model);
    }

    public Hardware() {
    }

    public String getBrand() {
        return Brand;
    }

    public void setBrand(String Brand) {
        this.Brand = Brand;
    }

    public String getModel() {
        return Model;
    }

    public void setModel(String Model) {
        this.Model = Model;
    }

    public void printHardware() {
        System.out.println("Brand : " + this.Brand);
        System.out.println("Model : " + this.Model);
    }
}

File: Python/Hardware.py
from Product import Product



class Hardware(Product):
    __Brand = ""
    __Model = ""
    
    def __init__(self, Brand = "", Model = ""):
        self.__Brand = Brand
        self.__Model = Model
        
    def setBrand(self, Brand):
        self.__Brand = Brand
        
    def getBrand(self):
        return self.__Brand
    
    def setModel(self, Model):
        self.__Model = Model
    
    def getModel(self):
        return self.__Model
    
    def printHardware(self):
        print("Brand            : " + str(self.__Brand))
        print("Model            : " + str(self.__Model))